.gitignore

```diff
@@ -40,3 +40,4 @@ coverage/
 ######################################
 !.gitignore
 !.gitkeep
+vendor
```

Comment:
I ran a bundle install I think. The vendor directory only contains ruby 2.1.0. I deleted it and removed that line from .gitignore.